pyproject.toml

```diff
@@ -56,6 +56,7 @@ dependencies = [
     "cons",
     "typing_extensions",
     "setuptools >=48.0.0",
+    "hypothesis",
```

Comment:
You might need to add this to the test set up script (i.e. [here](https://github.com/aesara-devs/aesara/blob/main/.github/workflows/test.yml#L126)).

Reply:
I added it. I'm not sure how to test it, as I'm a little fuzzy on what this script is used for, but I guess we'll find out when something happens on GitHub. :^)